docs/content/preview/admin/ysql-dumpall.md

```diff
@@ -75,11 +85,11 @@ Specify the superuser username to use when disabling triggers. This is relevant
 
 #### -v, --verbose
 
-Specifies verbose mode. This will cause `ysql_dumpall` to output start and stop times to the dump file, and progress messages to standard error. It will also enable verbose output in [`ysql_dump`](../ysql-dump/).
+Specifies verbose mode. This causes ysql_dumpall to output start and stop times to the dump file, and progress messages to standard error. It will also enable verbose output in [`ysql_dump`](../ysql-dump/).
```

Comment:
```suggestion
Specifies verbose mode. This causes ysql_dumpall to output start and stop times to the dump file, and progress messages to standard error. It will also enable verbose output in [ysql_dump](../ysql-dump/).
```

Reply:
remove the quotes for other instances of ysql_dump too.